flake.nix

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Hi, 
I use NixOS.  I will need to create a dev shell to develop.
This commit can be dropped after the PR is ready to merge, but it is good to let it in, and not affect the other PR. The other pack nix package with flake, but we are using different attributes, so it will not conflict.

Reply:
Yes, I know you need it, but I didn't really see the point in including it in the PR.
I think the important thing is that it has nothing to do directly with this PR itself.

Reply:
Okay, I revert it.